virt/virtual_machines/vm_networking/virt-configuring-ip-for-vms.adoc

```diff
@@ -10,9 +10,8 @@ You can configure either dynamically or statically provisioned IP addresses for
 
 .Prerequisites
 
-* The virtual machine can connect to a xref:../../../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#attaching-to-multiple-networks[secondary network]
+* The virtual machine can connect to a xref:../../../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-attaching-vm-multiple-networks[external network]
```

Comment:
a > an

The wording of the prerequisites should be parallel. The first one is "The VM can..." and the second is "you must have". It would be good to fix that so it is more consistent.

Reply:
Changed to:


.Prerequisites

* The virtual machine must connect to an xref:../../../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-attaching-vm-multiple-networks[external network]

* You must have a DHCP server available on the external network to configure a dynamic IP for the virtual machine.